src/types/Newsroom.ts

```diff
@@ -3,6 +3,8 @@ import type { UploadedImage } from '@prezly/uploads';
 import type { CultureRef } from './Culture';
 import type { NewroomThemeRef, NewsroomThemePreset } from './NewsroomTheme';
 
+type Iso8601DateTime = string;
```

Comment:
This type is already declared several times throughout the codebase, should we extract it as a global type and re-use it everywhere?

Reply:
> This type is already declared several times throughout the codebase, should we extract it as a global type and re-use it everywhere?

Yeah, probably :)